config/host/dev.exs

```diff
@@ -0,0 +1,8 @@
+import Config
+
+# Phoenix config:
+# Configures the endpoint
+config :xebow, XebowWeb.Endpoint,
+  http: [port: 4000],
+  url: [host: "localhost"],
+  code_reloader: true
```

Comment:
:no_entry: (required)

This is the rest of the Phoenix boilerplate I needed to copy in to get the live reload working. I also needed to change `"../../assets"` (added an extra `../`) since we nested the config one level deeper.

```suggestion
  debug_errors: true,
  code_reloader: true,
  check_origin: false,
  watchers: [
    node: [
      "node_modules/webpack/bin/webpack.js",
      "--mode",
      "development",
      "--watch-stdin",
      cd: Path.expand("../../assets", __DIR__)
    ]
  ]

# Watch static and templates for browser reloading.
config :xebow, XebowWeb.Endpoint,
  live_reload: [
    patterns: [
      ~r"priv/static/.*(js|css|png|jpeg|jpg|gif|svg)$",
      ~r"priv/gettext/.*(po)$",
      ~r"lib/intrinsic_web/(live|views)/.*(ex)$",
      ~r"lib/intrinsic_web/templates/.*(eex)$"
    ]
  ]

# Do not include metadata nor timestamps in development logs
config :logger, :console, format: "[$level] $message\n"

# Set a higher stacktrace during development. Avoid configuring such
# in production as building large stacktraces may be expensive.
config :phoenix, :stacktrace_depth, 20

# Initialize plugs at runtime for faster development compilation
config :phoenix, :plug_init_mode, :runtime
```

Reply:
Beautiful, thank you!